spec/support/flipper.rb

```diff
@@ -0,0 +1,13 @@
+module Flipper
+  RSpec.configure do |config|
+    config.before(:each) do |example|
+      allow(Panoptes).to receive(:flipper).and_return(Flipper.new(Flipper::Adapters::Memory.new))
+      Panoptes.flipper["cellect"].enable
+      Panoptes.flipper["designator"].enable
+      Panoptes.flipper[:remove_complete_subjects].enable
```

Comment:
are globally enabling these a good idea? I think the state is a little hidden in this file, the reactor makes sense so it's probably fine.